packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -302,6 +303,37 @@ export default {
             dispatch(`${VUEX_CHECKOUT_ANALYTICS_MODULE}/updateAutofill`, state, { root: true });
         },
 
+        /**
+         * Gets phone number from customer record from backend and updates state
+         *
+         * @param {Object} context - Vuex context object, this is the standard first parameter for actions
+         * @param {Object} payload - Parameter with the different configurations for the request.
+         */
+        getCustomer: async ({ commit, state, dispatch }, {
+            url,
+            timeout
+        }) => {
+            if (!state.customer || state.customer.mobileNumber) {
+                return;
+            }
```

Comment:
Isn't this validation redundant? Or in other words, if we have it here, do we need to validate this in other places?

Reply:
Will open separate PR